packages/modules/manage-versions/src/components/ManageVersions/ChangesTab/ChangesTab.scss

```diff
@@ -2,10 +2,9 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-@import "~@itwin/itwinui-css/scss/variables";
 
 .iac-changes-table {
-  .iui-table-body .iui-cell {
+  .iui-table-body .iui-table-cell {
```

Comment:
would be good to add class on cell instead than targeting iui classes

Reply:
I might be wrong but I don't see any way of adding custom class to cells or rows

Reply:
Have you tried adding `cellClassName` property on the column?

Reply:
oh, but in this case you need that class on all the columns, I assume? ahh.. probably doesnt matter then :/